src/univariate/discrete/dirac.jl

```diff
@@ -29,6 +29,9 @@ minimum(d::Dirac) = d.value
 maximum(d::Dirac) = d.value
 support(d::Dirac) = (d.value,)
 
+location(d::Dirac) = d.value
+scale(d::Dirac{T}) where {T} = one(T)
```

Comment:
Is this definition reasonable and useful at all? There's not really any scale to normalize or work with in the case of `Dirac`?

Reply:
Came here because I need this to work for distributions with parameter updates under rescaling (e.g. Normal, Uniform) and location-scale wrappers (e.g. Beta):
```
function _rescale_to_standart(X::UnivariateDistribution)
  m = location(X)
  s = scale(X)
  Y = 1.0/s * X - m
  return Y, mean(X), std(X)
end
```
Standart here is `Dirac(0.0)` and parametric form is preserved.

For the scale it's not 100% clear (see l.64), but in this case reasonable imho. I did not think of any counterexamples or other situations.